autoware_launch/launch/planning_simulator.launch.xml

```diff
@@ -81,13 +83,13 @@
     <let name="launch_dummy_perception" value="true" unless="$(var scenario_simulation)"/>
     <let name="launch_dummy_vehicle" value="false" if="$(var scenario_simulation)"/>
     <let name="launch_dummy_vehicle" value="true" unless="$(var scenario_simulation)"/>
-    <let name="launch_dummy_localization" value="true"/>
+    <let name="localization_sim_mode" value="$(var localization_sim_mode)"/>
```

Comment:
Now that we added localization_sim_mode as `arg` in the in the same file, I think we can remove this `let` statement.

Reply:
Sorry for my poor understanding. Thank you.